lib/ja_serializer/builder/included.ex

```diff
@@ -55,4 +62,23 @@ defmodule JaSerializer.Builder.Included do
   defp reject_known(resources, included, primary) do
     Enum.reject(resources, &(&1 in included || &1 in primary))
   end
+
+  defp normalize_relation_opts(opts, trigger_deprecation \\ false) do
```

Comment:
Can we move the deprecation and normalization to compile time in the `Serializer.has_many/2` and `Serializer.has_one/2` macros?